test/spec/functions.ts

```diff
@@ -48,7 +47,15 @@ function aggregate(input: string[]): string | null {
  */
 function aggregate_verify(input: {pubkeys: string[]; messages: string[]; signature: string}): boolean {
   const {pubkeys, messages, signature} = input;
-  return aggregateVerify(messages.map(fromHex), pubkeys.map(fromHex), fromHex(signature));
+  try {
+    return aggregateVerify(
+      messages.map(fromHex),
+      pubkeys.map((hex) => PublicKey.fromHex(hex)),
+      Signature.fromHex(signature)
+    );
+  } catch (e) {
+    return false;
+  }
```

Comment:
We are composing the functionality differently than we were before, converting strings/Uint8Arrays into point objects before passing them into verification functions.

There is no deviation from the spec, rather this library, and practical usage of bls by client teams, is a decomposition of the spec into pieces. In order to re-combine our functionality to the interface of the spec tests (which operates entirely on Uint8Arrays), we need this thin compatibility layer.

before:
```ts
function verify(msg: Uint8Array, pk: Uint8Array, sig: Uint8Array): boolean;

// when pk or sig deserializes to an invalid point, the verify function  returns false
verify(msg, badPk, badSig)
```

now:
```ts
function verify(msg: Uint8Array, pk: PublicKey, sig: Signature): boolean;

// when pk or sig deserializes to an invalid point, a PublicKey or Signature object cannot be created, verify cannot be attempted.
const pk = PublicKey.fromBytes(badPk); // this throws
const sig = Signature.fromBytes(badSig); // this throws
verify(msg, pk, sig); // this is never reached
```

Now, we _can_ and should tighten this compatibility layer to only catch known BLST errors rather than blanket catch errors.